custom-metrics-stackdriver-adapter/pkg/apiserver/apiserver.go

```diff
@@ -40,7 +41,8 @@ var (
 )
 
 func init() {
-	install.Install(groupFactoryRegistry, registry, Scheme)
+	cminstall.Install(groupFactoryRegistry, registry, Scheme)
+	eminstall.Install(groupFactoryRegistry, registry, Scheme)
```

Comment:
Shouldn't we only install api group if it's enabled?

Reply:
Do you mean to move everything from the init() function to completedConfig.New()? I can do that, but on the other hand, this only registers types and api group, so it doesn't have any effect visible for caller.